config/unix-g++.cmake

```diff
@@ -143,8 +148,10 @@ if( NOT CXX_FLAGS_INITIALIZED )
       # Wiki page for more details
       # (https://gcc.gnu.org/wiki/Intel%20MPX%20support%20in%20the%20GCC%20compiler)
 
-      # string( APPEND CMAKE_C_FLAGS_DEBUG
-      #         " -fsanitize=float-divide-by-zero -fcheck-ponter-bounds")
+      string( APPEND CMAKE_C_FLAGS_DEBUG
+        " -fsanitize=float-divide-by-zero -fcheck-ponter-bounds")
```

Comment:
Additionally, `-fcheck-pointer-bounds` isn't supported on any of our machine.  I'm going to rework the logic a bit and turn on `-fsanitize=float-cast-overflow` instead.